src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1611,7 +1623,8 @@ public static async ValueTask NotifyFeaturesAboutStateChangeAsync(DatabaseRecord
                             }
                             catch (Exception e)
                             {
-                                state.Logger.Operations($"Failed to log long held cluster lock: {sp.Elapsed} in database {state.Name}", e);
+                                if (state.Logger.IsDebugEnabled)
+                                    state.Logger.Debug($"Failed to log long held cluster lock: {sp.Elapsed} in database {state.Name}", e);
```

Comment:
Given that we're in `if (... state.Logger.IsWarnEnabled)` scope, I believe the exception should also be logged at warning level.